install_gcloud.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#Install gcloud
+if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
```

Comment:
So this script is copied from a google repo.  Not sure what the legality of that is.  Can we have a google person bless it?